src/client/mod.rs

```diff
@@ -193,8 +192,9 @@ impl<RequestCtx: RequestContext> Client<RequestCtx> {
             Client::Local { engine, metadata } => {
                 engine
                     .execute(
```

Comment:
Changed the signature of execute to take discrete arguments instead of requiring a juniper-specific struct (juniper::GraphQLRequest).